### ISSUE TITLE
Como estimar a quantidade de pessoas expostas?

### ISSUE THREAD

**Issue by dsevero** [3778/COVID-19#43]
No momento, estamos assumindo que a quantidade de expostas e infectadas sao a mesma (E(0) = I(0)).

Nao tenho ideia se isso ta certo, alguem saberia ajudar? Tem alguma estimativa grosseira pra quantidade de pessoas expostas em ( talvez em funcao de infectadas)?

**Comment by victorlevi94** [comment]
Não sei se entendi bem ou se estou sendo altamente ignorante acerca da modelagem, mas S(0) não seria N - I(0) - R(0) ?

**Comment by dsevero** [comment]
Sim, voce esta certo. Porem, estamos usando um modelo SEIR, nao SIR. A pergunta eh sobre o compartimento E.

**Comment by vksvinicius** [comment]
Eu fiz algumas pequenas modificacoes nas equacoes para tratar do caso no início do surto.

Dividindo as equações pelo total da população N, chega-se as seguintes equações:
![image](https://user-images.githubusercontent.com/35741807/77214750-77001700-6ade-11ea-8837-5f8244b93721.png)

Integrando num tempo curto:
![image](https://user-images.githubusercontent.com/35741807/77214866-132a1e00-6adf-11ea-8f38-4ed6bc920808.png)

No tempo t=0, s=1 e os outros compartimentos são praticamente nulos. Fazendo uma interpolação linear no tempo:
![image](https://user-images.githubusercontent.com/35741807/77214899-47054380-6adf-11ea-9e03-f537c61952ed.png)
Dividindo as equações chega em:
![image](https://user-images.githubusercontent.com/35741807/77214915-55ebf600-6adf-11ea-9151-d7bd46ab2b62.png)
Que pode ser resolvido para a razão de Expostos e Infectados:
![image](https://user-images.githubusercontent.com/35741807/77214936-7025d400-6adf-11ea-97cd-fed8054a43bc.png)

Essa equação poderia ser utilizada como primeira aproximação pro valor de E baseado em I.

**Comment by vksvinicius** [comment]
Acho que um jeito mais fácil de chegar na equação é dividir as duas equações:
![image](https://user-images.githubusercontent.com/35741807/77215211-e971f680-6ae0-11ea-9045-db14b63a6fbf.png)
Que chega na solução
![image](https://user-images.githubusercontent.com/35741807/77215226-00b0e400-6ae1-11ea-9aee-ab901bac95c6.png)

**Comment by tiagoyuzo** [comment]
Pra mim o raciocínio faz sentido (não refiz os cálculos). É possível usar para pelo menos ter um racional como justificativa.

**Comment by Mauroncard** [comment]
Essa parte, pelas brincadeiras que fiz, parece mudar pouco os resultados, mas acho estranho o E(0) mais baixo que o I(0) no começo pois a exposição precede a infecção. É normal o E(0) cair depois se a gente colocar um período infeccioso mais que o período de exposição. Se a gente está considerando um R0 maior que 1, então o E(0) poderia ser proporcional ao I(0) em um valor proprocorcional a esse R0. Algo como o dobro ou o tripo.

**Comment by dsevero** [comment]
Então, o E(0) tem efeito principalmente no começo da curva. Note que, se colocarmos o E(0) < I(0) temos que I(t) primeiro diminui pra depois aumentar, por uns 2 ou 3 dias. Isso é bem estranho, e com ctz não é um resultado que queremos. Eu to colocando E(0) = 2*I(0) por hora.
